src/components/tab/tab.module.css

```diff
@@ -0,0 +1,74 @@
+@value colors: "../../css/colors.css";
+@value primaryColor from colors;
+
+.tabList {
+  height: 48px;
+  background-color: #ffffff;
+  display: flex;
+  justify-content: space-around;
+  box-shadow: inset 0px -1px 0px rgba(208, 201, 214, 0.4);
+  font-family: Avenir;
+}
+
+.tabsTab {
+  height: 100%;
+  position: relative;
+}
+
+.tabsTab:active,
+.tabsTab:focus {
+  outline: none;
+}
+
+.tabContainer {
+  position: relative;
+  height: 48px;
+  padding: 0;
+  cursor: pointer;
+}
+
+.tabContainerDisabled {
+  cursor: default;
+  opacity: 0.7;
+}
+
+.tabTextContainer {
+  position: relative;
+  height: 100%;
+  width: 100%;
+  color: rgba(40, 55, 86, 0.33);
+  display: flex;
+  text-transform: uppercase;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  font-size: 14px;
+  /* Without this; Safari briefly fades the text color whilst ::active.*/
+  -webkit-fontsmoothing: subpixel-antialiased;
+}
+
+.tabContainer:hover {
```

Comment:
Can you add some left and right padding to the hover states so the text stands out from the hover color?

Reply:
Oh yeah. Okay